test-e2e/test_e2e_module.py

```diff
@@ -9,6 +10,17 @@
 from dcos_e2e.backends import Docker
 
 
+@pytest.fixture(scope='session', autouse=True)
+def configure_logging() -> None:
+    """
+    Surpress DEBUG log messages from libraries that log excessive amount of
+    debug output that isn't useful for debugging e2e tests.
```

Comment:
the comment is out of sync with code. We suppress INFO level, too.

:man_shrugging:

Reply:
Yes you are right - good spot - given that Martin's original has already been merged to `master`, and that we are happy with the code (therefore only unhappy with the comment) and that this is in 8 places (OSS + EE x 4 versions) I'm going to opt not to make any change. Please let me know if you don't agree with this decision.